rllib/utils/checkpoints.py

```diff
@@ -0,0 +1,121 @@
+import os
+import tempfile
+import re
+from typing import Any, Dict
+
+from ray.air.checkpoint import Checkpoint
+
+# The current checkpoint version used by RLlib for Algorithm and Policy checkpoints.
+# History:
+# v0: Ray 2.0.0
+#  A single `checkpoint-[iter num]` file for Algorithm checkpoints
+#  within the checkpoint directory. Policy checkpoints not supported across all
+#  DL frameworks.
+
+# v1: Ray >=2.1.0
+# TODO:
+CHECKPOINT_VERSION = "v1"
```

Comment:
what do you think if we keep ``CHECKPOINT_VERSION`` an integer, instead of string "v0"?
this is so we don't have to add special logics for ``"v11" > "v2"`` down the road.

Reply:
Using `packaging.version` now.